### PR TITLE
Pass the "npa" field from publisher's app to AdMob via ad request

### PR DESCRIPTION
**Pull request by chauduyphanvu1**

### CODE REVIEW

File: AdMob/com/mopub/mobileads/GooglePlayServicesInterstitial.java [hunk 1]
```diff
@@ -67,14 +71,25 @@ protected void onInvalidate() {
         }
     }
 
+    private Bundle getConsent(Map<String, Object> localExtras) {
```
Comment (rsatrasala) [inline AdMob/com/mopub/mobileads/GooglePlayServicesInterstitial.java]: Rename this method to getGooglePersonalizationPreferences. 

Add a comment that this setting is applicable to only publishers who use their own consent mechanism. Consent collected from MoPub's default consent dialogue should not used to used to pass the value of npa here

File: AdMob/com/mopub/mobileads/GooglePlayServicesRewardedVideo.java [hunk 2]
```diff
@@ -167,7 +171,7 @@ public void run() {
                             .onRewardedVideoLoadSuccess(GooglePlayServicesRewardedVideo.class, mAdUnitId);
                 } else {
                     mRewardedVideoAd
-                            .loadAd(mAdUnitId, new AdRequest.Builder().setRequestAgent("MoPub").build());
+                            .loadAd(mAdUnitId, new AdRequest.Builder().addNetworkExtrasBundle(AdMobAdapter.class, getConsent(localExtrasMap)).setRequestAgent("MoPub").build());
```
Comment (rsatrasala) [inline AdMob/com/mopub/mobileads/GooglePlayServicesRewardedVideo.java]: check is localExtrasMap is empty

Comment (chauduyphanvu1) [reply AdMob/com/mopub/mobileads/GooglePlayServicesRewardedVideo.java]: It's checked below in the TextUtils.isEmpty() line.

Comment (chauduyphanvu1) [reply AdMob/com/mopub/mobileads/GooglePlayServicesRewardedVideo.java]: Also within the Android SDK, `localExtras` is passed around either as a `@NonNull` or populated with a few other key-value pairs, so it is null-safe. The only thing that might be null/empty is the `npa`, and it's being guarded in this PR.

File: AdMob/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
 ## Changelog
+  * 15.0.0.4
+    * Pass ad personalization preference to AdMob as per GDPR. Unlike with the other mediation adapters, publishers must specify their preference ("npa") in their app implementation. The adapter will read from a MoPub's `localExtras` and forward that onto AdMob via an AdMob ad request.
```
Comment (rsatrasala) [inline AdMob/CHANGELOG.md]: Simplify - Append user's ad personalization preference from`localExtras` to AdMob's ad request.

File: AdMob/com/mopub/mobileads/GooglePlayServicesBanner.java [hunk 4]
```diff
@@ -62,6 +65,10 @@ protected void loadBanner(
 
         final AdRequest adRequest = new AdRequest.Builder()
                 .setRequestAgent("MoPub")
+                // This setting is applicable to only publishers who use their own consent mechanism.
```
Comment (rsatrasala) [inline AdMob/com/mopub/mobileads/GooglePlayServicesBanner.java]: Consent collected from the MoPub’s consent dialogue  should not be used to set up Google's personalization preference. Publishers should work with Google to be GDPR-compliant.

**Comment by rsatrasala** [comment]
LGTM with the comment changes

**Pull request merged**